src/api/api.service.ts

```diff
@@ -1,22 +1,36 @@
-import { Injectable } from '@nestjs/common';
+import { Injectable, NotAcceptableException } from '@nestjs/common';
 import { BullService } from 'src/bull/bull.service';
 import { PrismaService } from 'src/prisma/prisma.service';
 import { SearchResult } from './types';
 import { Job } from 'bullmq';
 
 @Injectable()
 export class ApiService {
+  private keyword_limit = 100;
```

Comment:
This could be defined as a constant or even environment variable to make It easy to change this value